pymc_marketing/mmm/multidimensional.py

```diff
@@ -1057,7 +1057,7 @@ def build_model(
                     dims=("date", *self.dims, "channel"),
                 )
 
-                media_latent_process = HSGP.parameterize_from_data(
+                media_latent_process = SoftPlusHSGP.parameterize_from_data(
```

Comment:
@wd60622 can you please explain the motivation behind the softplus transform? 🙏

Reply:
- Aligns the default hsgp with other MMM class
- remove negative channel contributions with time varying media = True

Reply:
Thanks @wd60622 ! Would you mind running the time-varying parameter notebooks affected by this change just to control the expected result? 🙏

Reply:
Is there a multidimensional mmm notebook already? This is just the new class

Reply:
Right, but in theory you could reproduce the results of the MMM basic class with this object right? The intention is not to update the docs but maybe have some experiments on the effect of this parametrization. I think you can just share some plots or experiments 9for the sake of illustrating the benefits) :) Just for future reference. It is not a blocker, but more to have a bit more intuition WDYT?

Reply:
Sure. I can update 
However, there is still slight difference in the intercept which I noted. Should that be consolidated? 

Thoughts @cetagostini